patch-package
--- a/node_modules/karma-webpack/lib/karma-webpack.js
+++ b/node_modules/karma-webpack/lib/karma-webpack.js
@@ -74,7 +74,7 @@ function Plugin(
     // Must have the common _karma_webpack_ prefix on path here to avoid
     // https://github.com/webpack/webpack/issues/645
     webpackOptions.output.path = path.join(os.tmpdir(), '_karma_webpack_', indexPath, '/');
-    webpackOptions.output.publicPath = path.join(os.tmpdir(), '_karma_webpack_', publicPath, '/');
+    webpackOptions.output.publicPath = path.join('/_karma_webpack_', publicPath, '/');
     webpackOptions.output.filename = '[name]';
     if (includeIndex) {
       webpackOptions.output.jsonpFunction = `webpackJsonp${index}`;
@@ -198,7 +198,7 @@ function Plugin(
     compiler.plugin('invalid', invalid.bind(this));
   }
 
-  webpackMiddlewareOptions.publicPath = path.join(os.tmpdir(), '_karma_webpack_', '/');
+  webpackMiddlewareOptions.publicPath = '/_karma_webpack_/';
   var middleware = this.middleware = new webpackDevMiddleware(compiler, webpackMiddlewareOptions);
 
   customFileHandlers.push({
